library/iterator.lisp

```diff
@@ -496,6 +520,9 @@ The vector will be resized if ITER contains more than SIZE elements."
     "Construct a `Vector` containing all the elements from ITER in order."
     (collect-vector-size-hint! 0 iter))
 
+  (define-instance (types:RuntimeRepr :elt => FromIterator (Vector :elt) :elt)
+    (define collect! collect-vector!))
```

Comment:
this looks like a bug

Reply:
what looks like a bug here?

Reply:
Oh, oops, I misread the definition. No bug.